rendezvousService.sh

```diff
@@ -5,6 +5,7 @@
 
 java -Dserver.ssl.key-store=./certs/rendezvous-keystore.jks -Dserver.ssl.key-store-password=123456 \
      -Djavax.net.ssl.trustStore=./certs/rendezvous-trusterRootCA.jks -Djavax.net.ssl.trustStorePassword=123456 \
+     -Drendezvous.hmacSecret=$(tr -cd 'a-f0-9' < /dev/urandom | head -c128) \
```

Comment:
/dev/urandom is less secure than /dev/random.   Please change it to /dev/random.